scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -67,6 +67,11 @@ ruleTester.run('protractor-practices', rule, {
     code: 'var modal = element.all(by.css(".modal-dialog")).last();'
   }, {
     code: 'var modal = element(by.css("option:checked"));'
+  }, {
+    code: ` let items = [];
+            for(let i=0;i< items.length; i++){
+            console.log(items[i])};
```

Comment:
Can you clean up the spacing and indentation here? It should be just as clean and readable as real code